tests/test/org/commcare/cases/test/CaseParseAndReadTest.java

```diff
@@ -41,7 +41,7 @@ public void testReadCaseDB() throws Exception {
 
     private void compareCaseDbState(String inputTransactions,
                                     String caseDbState) throws Exception {
-        MockDataUtils.parseIntoSandbox(this.getClass().getResourceAsStream(inputTransactions), sandbox);
+        MockDataUtils.parseIntoSandbox(this.getClass().getResourceAsStream(inputTransactions), sandbox, true);
```

Comment:
Since ```parseIntoSandbox``` is overridden to set ```failfast``` to true if it isn't provided, you shouldn't need to update existing calls to it, unless you want ```failfast``` set to false.